crates/sdf-http/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sdf-http"
```

Comment:
shouldn't we use `sdf-http-guest`?

Reply:
let's stick for now.  we have to change all names.  besides, all client libraries are similar `http`